test/Go/GoListSpec.hs

```diff
@@ -69,7 +70,9 @@ spec = do
               & run
       case result of
         Left err -> expectationFailure ("analyze failed: " <> show (renderFailureBundle err))
-        Right graph -> graph `shouldBe` expected
+        Right (graph, graphType) -> do
+          graph `shouldBe` expected
+          graphType `shouldBe` Complete
```

Comment:
[nit]: I don't think it's very useful to test the graph breadth, it's just a constant value
```suggestion
        Right (graph, _) -> graph `shouldBe` expected
```